File: baseProjectPY/pipelines/basePipeline1/code/basepipeline1/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from basepipeline1.config.ConfigStore import *
from basepipeline1.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        udfConcat(col("first_name"), col("last_name")).alias("name"), 
        udfConcatold(col("first_name"), col("last_name")).alias("name2")
    )

File: baseProjectRW/pipelines/basePipeline1/code/src/main/scala/io/prophecy/pipelines/basepipeline1/graph/Subgraph_1/Reformat_1.scala
package io.prophecy.pipelines.basepipeline1.graph.Subgraph_1

import io.prophecy.libs._
import io.prophecy.pipelines.basepipeline1.udfs.UDFs._
import io.prophecy.pipelines.basepipeline1.graph.Subgraph_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(
      createFullName(col("first_name"), col("last_name")).as("full_name")
    )

}
